commerce_coordinator/apps/rollout/pipeline.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+import waffle
+from openedx_filters import PipelineStep
+
+from commerce_coordinator.apps.commercetools_frontend.constants import COMMERCETOOLS_FRONTEND
+from commerce_coordinator.apps.frontend_app_payment.constants import FRONTEND_APP_PAYMENT_CHECKOUT
+from commerce_coordinator.apps.rollout.constants import REDIRECT_TO_COMMERCETOOLS_CHECKOUT
+
+logger = logging.getLogger(__name__)
+
+
+class GetActiveOrderManagementSystem(PipelineStep):
+    """
+    Determines which order management system to redirect to, based on enterprise role.
```

Comment:
Can we update the description to reflect that order management is determined based on the existence of course_run_id and waffle flag's value?